checkov/cdk/checks_infra/base_registry.py

```diff
@@ -18,3 +18,9 @@ def load_rules(
         self, frameworks: Iterable[str], sast_languages: set[SastLanguages] | None
     ) -> int:
         return self._load_checks_from_dir(directory=self.checks_dir, sast_languages=SastLanguages.set())
+
+    def load_semgrep_checks(self, languages: set[SastLanguages]) -> int:
```

Comment:
Is this a duplication of the method in `checkov/sast/checks_infra/base_registry.py`? Do we need them both?

Reply:
yes we do, this is the only way that unitest work's. the logic inside changed